src/ontology/edits/oeo-shared.omn

```diff
@@ -1576,6 +1579,34 @@ pull request: https://github.com/OpenEnergyPlatform/ontology/pull/1652",
         <http://purl.obolibrary.org/obo/BFO_0000023>
     
     
+Class: OEO_00010489
+
+    Annotations: 
+        <http://purl.obolibrary.org/obo/IAO_0000115> "A unit of measurement is a standardised quantity of a quantifiable entity"@en,
+        <http://purl.obolibrary.org/obo/IAO_0000118> "Einheit"@de,
+        <http://purl.obolibrary.org/obo/IAO_0000233> "issue: https://github.com/OpenEnergyPlatform/ontology/issues/1871
+pull request: https://github.com/OpenEnergyPlatform/ontology/pull/1892",
+        rdfs:label "unit"@en
+    
+    SubClassOf: 
+        <http://purl.obolibrary.org/obo/BFO_0000031>
+    
+    
+Class: OEO_00010490
+
+    Annotations: 
+        <http://purl.obolibrary.org/obo/IAO_0000118> "physikalische Einheit"@en,
```

Comment:
Based on the def of uo:unit, which is _A unit of measurement is a standardized quantity of a physical quality_ I'd suggest:

```suggestion
        <http://purl.obolibrary.org/obo/IAO_0000118> "physikalische Einheit"@en,
        <http://purl.obolibrary.org/obo/IAO_0000115> "A physical unit s a unit that represents a standardised quantity value of a physical quantifiable entity"@en,
```